twilio/http/http_client.py

```diff
@@ -47,4 +72,12 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
             timeout=timeout,
         )
 
+        if hasattr(response, 'json') and response.json():
+            # strip JSON of possible prettyprint
```

Comment:
The API doesn't pretty print. We probably want to save the `loads`/`dumps` cost by just always using `response.text`.